core/cmd/remote_client.go

```diff
@@ -564,3 +565,44 @@ func (cli *Client) CreateExtraKey(c *clipkg.Context) error {
 
 	return cli.printResponseBody(resp)
 }
+
+// SetMinimumGasPrice specifies the minimum gas price to use for outgoing transactions
+func (cli *Client) SetMinimumGasPrice(c *clipkg.Context) error {
+	if c.NArg() != 1 {
+		return cli.errorOut(errors.New("expecting an amount"))
+	}
+
+	value := c.Args().Get(0)
+	amount, ok := new(big.Float).SetString(value)
+	if !ok {
+		return cli.errorOut(fmt.Errorf("invalid ethereum amount %s", value))
+	}
+
+	if c.IsSet("gwei") {
+		amount.Mul(amount, big.NewFloat(1000000000))
+	}
+
+	adjustedAmount, _ := amount.Int(nil)
+	request := struct {
+		EthGasPriceDefault string `json:"ethGasPriceDefault"`
+	}{EthGasPriceDefault: adjustedAmount.String()}
+	requestData, err := json.Marshal(request)
+	if err != nil {
+		return cli.errorOut(err)
+	}
+
+	buf := bytes.NewBuffer(requestData)
+	response, err := cli.HTTP.Patch("/v2/config", buf)
+	if err != nil {
+		return cli.errorOut(err)
+	}
+	defer response.Body.Close()
+
+	patchResponse := web.ConfigPatchResponse{}
+	if err := cli.deserializeAPIResponse(response, &patchResponse, &jsonapi.Links{}); err != nil {
+		return err
+	}
+
+	fmt.Printf("EthGasPriceDefault updated %s -> %s.\n", patchResponse.OldEthGasPriceDefault, patchResponse.NewEthGasPriceDefault)
```

Comment:
Do we want `fmt.Printf` or do we want the `cli.Renderer` to output the result? I believe the goal was to route things through renderer because it supports both tabular output and json, and is easier to test. Looks like over time we've added a few `fmt.Print...` statements in the remote client, since they don't really fit the mold of a table or key value in json.

Routing the fmt.Prints through `Renderer` might be for another PR, if at all needed.

Reply:
Nah I can see a way to format this in a tabular way that would be nice.